CHANGELOG_DEV.md

```diff
@@ -1,3 +1,9 @@
+## 0.85.0 - 2020-12-08
+
+- Extended TLS support and mapping to clientID for client's key selection purposes
+  - Strategy of extraction metadata from certificates for mapping to clientID: `tls_identifier_extractor_type` (default: `distinguished_name` and `serial_number`)
```

Comment:
I guess we need to tell users that `and 'serial_number'` is another possible value, and it has nothing to do with default one, so maybe tell explicitly that `serial_number` is a completely different option?
```suggestion
  - Strategy of extraction metadata from certificates for mapping to clientID: `tls_identifier_extractor_type` (default: `distinguished_name`, another option: `serial_number`)
```

Reply:
it's just changelog_dev, not public cli description of param. but okay)